### PR TITLE
OpenLane tag update to 2022.07.26

### PR DESCRIPTION
**Pull request by vijayank88**
OpenLane tag update to [2022.07.21](https://github.com/The-OpenROAD-Project/OpenLane/releases/tag/2022.07.21)

`script_dir` deprecated and updated to `set script_dir $::env(DESIGN_DIR)`

So updated `source openlane/top/config.tcl` to `source $::env(DESIGN_DIR)/openlane/top/config.tcl` to fix following error

```
[INFO]: Using configuration in 'designs/AC_top/config.tcl'...
couldn't read file "openlane/top/config.tcl": no such file or directory
    while executing
"source openlane/top/config.tcl"
```

### CODE REVIEW

**Comment by vijayank88** [comment]
@vvbandeira 
Fixed most of configuration updates wrt., latest OpenLane. Please approve and merge once its green.
Some of unstable design passed with latest OpenLane. After merging this will move to stable CI list. Thanks.

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**
